codegen/src/main/java/software/amazon/awssdk/codegen/AddMetadata.java

```diff
@@ -104,7 +104,8 @@ private static void configurePackageName(Metadata metadata,
                 .withPaginatorsPackageName(namingStrategy.getPaginatorsPackageName(service))
                 .withWaitersPackageName(namingStrategy.getWaitersPackageName(service))
                 .withEndpointRulesPackageName(namingStrategy.getEndpointRulesPackageName(service))
-                .withAuthSchemePackageName(namingStrategy.getAuthSchemePackageName(service));
+                .withAuthSchemePackageName(namingStrategy.getAuthSchemePackageName(service))
+                .withJPathPackageName(namingStrategy.getJPathPackageName(service));
```

Comment:
Should we use the full name `JmesPath` to be more clear?

Reply:
I explicitly avoided JmesPath since there has been a lot of discussion around minified versions, not calling that subset JmesPath etc. However, after working with the implementation in Java SDK that distinction has faded - I think the subset will be handled at a model level instead. So if consensus is JmesPath I'm fine with that.

Reply:
changed